test/ctia/encryption/default_test.clj

```diff
@@ -1,23 +1,29 @@
 (ns ctia.encryption.default-test
   (:require
-   [ctia.encryption :as enc]
    [ctia.encryption.default :as sut]
-   [clojure.test :as t :refer [deftest testing is]]))
+   [ctia.test-helpers.core :as helpers]
+   [clojure.test :as t :refer [deftest testing is]]
+   [puppetlabs.trapperkeeper.testutils.bootstrap :refer [with-app-with-config]]))
 
 (deftest encryption-default-record-test
-  (let [key-path "resources/cert/ctia-encryption.key"
-        rec (sut/map->EncryptionService
-             {:state (atom nil)})]
-    (testing "service init"
-      (try (enc/init rec {})
-           (catch Throwable e
-             (is (= (.getMessage e)
-                    "Assert failed: no secret or key filepath provided\n(or secret (:filepath key))"))))
-      (enc/init rec {:key {:filepath key-path}})
+  (let [key-path "resources/cert/ctia-encryption.key"]
+    (testing "failed service init"
+      (assert
```

Comment:
Not sure if I like the assert wrapping

Replacing `false` with `(assert false)` would be equivalent, while reducing nesting and avoiding the `(assert (is ...))` path which seems to rely on the `is` ret val (which I don't think is an API)

Reply:
Replaced with `thrown-with-msg?`.